201-front-door-managed-waf-ruleset/azuredeploy.json

```diff
@@ -20,147 +20,169 @@
+                                        "properties": {
+                                            "path": "/",
+                                            "protocol": "Http",
+                                            "intervalInSeconds": 120
+                                        }
+                                    }
+                                ],
+                                "loadBalancingSettings": [
+                                    {
+                                        "name": "loadBalancingSettings1",
+                                        "properties": {
+                                            "sampleSize": 4,
+                                            "successfulSamplesRequired": 2,
+                                            "additionalLatencyMilliseconds": 0
+                                        }
+                                    }
+                                ],
+                                "backendPools": [
+                                    {
+                                        "name": "backendPool1",
+                                        "properties": {
+                                            "backends": [
+                                                {
+                                                    "address": "[parameters('backendAddress')]",
+                                                    "backendHostHeader": "[parameters('backendAddress')]",
+                                                    "httpPort": 80,
+                                                    "httpsPort": 443,
+                                                    "weight": 1,
+                                                    "priority": 1,
+                                                    "enabledState": "Enabled"
+                                                }
+                                            ],
+                                            "loadBalancingSettings": {
+                                                "id": "[resourceId('Microsoft.Network/frontDoors/loadBalancingSettings', parameters('frontDoorName'), 'loadBalancingSettings1')]"
+                                            },
+                                            "healthProbeSettings": {
+                                                "id": "[resourceId('Microsoft.Network/frontDoors/healthProbeSettings', parameters('frontDoorName'), 'healthProbeSettings1')]"
+                                            }
+                                        }
+                                    }
+                                ],
+                                "frontendEndpoints": [
+                                    {
+                                        "name": "frontendEndpoint1",
+                                        "properties": {
+                                            "hostName": "[concat(parameters('frontDoorName'), '.azurefd.net')]",
```

Comment:
what happens if this host name is used in Gov cloud or Mooncake?

Reply:
AFD and WAF are not yet supported on any of the sovereign clouds.

Reply:
Ok, but when they are, this won't work...

Reply:
Updated template to use a variable instead of hardcoding value within the Front Door resource.